README.md

```diff
@@ -71,6 +71,16 @@ Each row represents an input genome and the fields are:
 - `fasta:` fasta file for the genome
 - `is_masked`: yes or no to denote whether the fasta file is already masked or not
 
+#### `--min_contig_length`
```

Comment:
Parameter documentation is auto generated with the following command,

```bash
nf-core -v pipelines schema docs > docs/parameters.md
```

The parameters are documented in the `docs/parameters.md` file.